### PR TITLE
Fix awkward sentence

### PR DESCRIPTION
**Pull request by lilianabiven**
## Checklist

- [x] Lint and unit tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added documentation to describe my feature.
- [x] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -44,7 +44,7 @@ translations (you can do it in :guilabel:`Repository maintenance` in the
 on).
 
 If you've already ran into a merge conflict, the easiest way is to solve all
-conflicts locally at your workstation - is to simply add Weblate as a remote
+conflicts locally at your workstation : simply add Weblate as a remote
```
Comment (nijel) [inline docs/faq.rst:47]: This reverts text before 9531c2a4920c7293bbf44acb0389f86f80d8ac0e (see https://github.com/WeblateOrg/weblate/pull/3178) by @comradekingu

Comment (comradekingu) [reply docs/faq.rst:47]: That should have been an em dash. The colon is an error to.
Original sentence with a comma works.

Comment (comradekingu) [inline docs/faq.rst:47]:
```suggestion
conflicts locally on your machine, is to add Weblate as a remote
```

Comment (lilianabiven) [reply docs/faq.rst:47]: So : "the easiest way **is to** solve all conflicts locally on your machine, **is to** add Weblate as a remote"?
That sentence seems incorrect to me, but I'm not a native speaker

File: docs/faq.rst [hunk 2]
```diff
@@ -44,7 +44,7 @@ translations (you can do it in :guilabel:`Repository maintenance` in the
 on).
 
 If you've already ran into a merge conflict, the easiest way is to solve all
```
Comment (comradekingu) [inline docs/faq.rst:46]:
```suggestion
If you've already ran into a merge conflict, the easiest way to solve all
```

**Comment by comradekingu** [comment]
This is solid.

**Pull request merged**